.github/workflows/luajit-integration.yml

```diff
@@ -0,0 +1,86 @@
+# This workflow is used within tarantool/luajit repository CI to
+# implement integration testing without manual LuaJIT submodule
+# bump and draft PR creation.
+# It bumps the given LuaJIT revision for the given Tarantool
+# release, builds Tarantool with the given buildtype and GC64
+# option and runs Tarantool testing suite.
+
+name: LuaJIT integration testing
+
+on:
+  workflow_call:
+    inputs:
+      buildtype:
+        description: CMake build type value
+        required: false
+        type: string
+        default: Debug
+      GC64:
+        description: CMake option value to enable LuaJIT GC64 mode
+        required: false
+        type: string
+        default: OFF
+      host:
+        description: Type of machine to run the GitHub job on
+        required: true
+        type: string
+      release:
+        description: Git revision from tarantool/tarantool repository
+        required: false
+        type: string
+        default: master
+      revision:
+        description: Git revision from tarantool/luajit repository
+        required: true
+        type: string
+
+jobs:
+  luajit-integration:
+    runs-on: ${{ inputs.host }}
+    steps:
+      - name: Prepare workspace
+        uses: tarantool/actions/cleanup@master
```

Comment:
It would be good to separate steps by an empty line for more readability.

Reply:
Fixed.